### PR TITLE
Added support for phrase search

### PR DESCRIPTION
**Pull request by XinyiYan**
- Use double quotes for Phrase search
- Use space to separate individual keyword
- Support a mixture of phrase and separate words search

Example:
Phrase search:   "vitamin d"
Separate words search: coronavirus cure
Mixture: "vitamin d"  coronavirus cure

### CODE REVIEW

File: web/Web/web/static/js/highlight.js [hunk 1]
```diff
@@ -182,6 +182,59 @@ $(function() {
     marked_matches_counter = {};
   }
 
+  /** This function processes the entered keyword to support both
```
Comment (ammsa) [inline web/Web/web/static/js/highlight.js:185]: Minor edit. Just to follow standard commenting style, please add new line after `/**`

**Comment by ammsa** [comment]
Seems to be working. What happens if `"` is part of the phrase I'm searching for?

File: web/Web/web/static/js/highlight.js (hunk 1 above)
Comment (ammsa) [reply web/Web/web/static/js/highlight.js:185]: @XinyiYan Can you edit this? I'll merge this pull request right after.

Comment (XinyiYan) [reply web/Web/web/static/js/highlight.js]: Done.

**Comment by ammsa** [comment]
Thank you! Merging.

**Pull request merged**